src/uproot/writing/identify.py

```diff
@@ -735,12 +735,14 @@ def to_TObjString(string):
     This function is for developers to create TObjString objects that can be
     written to ROOT files, to implement conversion routines.
     """
+    tobject = uproot.models.TObject.Model_TObject.empty()
+
     tobjstring = uproot.models.TObjString.Model_TObjString(str(string))
     tobjstring._deeply_writable = True
     tobjstring._cursor = None
     tobjstring._parent = None
     tobjstring._members = {}
-    tobjstring._bases = (uproot.models.TObject.Model_TObject(),)
+    tobjstring._bases = [tobject]
```

Comment:
Let's just keep this as a tuple. It's probably a tuple everywhere else.

```suggestion
    tobjstring._bases = (tobject,)
```

It _might_ matter if there's some code that concatenates lists of bases with `+` (`[1] + (2, 3)` is an error), though I don't think that there is. Still, if we just stay consistent with these types, the problem will never come up.